src/Components/Form/Form.js

```diff
@@ -10,22 +10,43 @@ const POST_REVIEW = gql`
     $description: String!
     $lat: String!
     $lng: String!
+    $dairyFree: Int = 0
```

Comment:
Is there a reason dairyFree has a `= 0` after `Int` and the others don't?

Reply:
Yeah, that was a leftover piece from the trial and error with Weston! It's removed now. Good catch!